packages/story-editor/src/components/canvas/karma/cloneSelection.karma.js

```diff
@@ -94,12 +94,11 @@ describe('Clone element integration', () => {
   });
 
   it('should correctly clone 1 element', async () => {
-    // Select img1
-    await clickElement(img1.id);
+    // Select img2
+    await clickElement(img2.id);
```

Comment:
Hm, this doesn't seem related to this PR though, but if it fixes things... 🤷

Reply:
The main change should in theory be in the `clickElement`, changing the clicking location (the element didn't get selected properly since the selection frame was a bit shifted in the tests -- not sure why it stopped working now in this PR, I think this shifted selection frame issue we've had previously already in the tests, IIRC)